pkg/jobs/registry.go

```diff
@@ -1307,6 +1307,10 @@ func (r *Registry) stepThroughStateMachine(
 			return errors.Errorf("job %d: node liveness error: restarting in background", job.ID())
 		}
 
+		if pauseOnErrErr := r.CheckPausepoint("error"); pauseOnErrErr != nil {
```

Comment:
My favourite bikeshed colour is "resume-error'

Reply:
I thought about something like that, but felt like it might be confusing or at least pretty non-obvious to anyone except those steeped in the internals and terminology of the jobs system. I was tempted to do "exec-error" or "on-error" but couldn't come up with what either of those offered over just "error" (we don't prefix the other conditions with "on" in other pause points)

Reply:
renamed to `after_exec_error`. WDYT?